src/steam.cpp

```diff
@@ -645,6 +651,12 @@ void steamAchievementClient(int player, const char* achName)
 		return;
 	}
 
+	if ( !(svFlags & SV_FLAG_CHEATS) )
```

Comment:
if ( !(svFlags & SV_FLAG_CHEATS) ) { //code } should be
if ( (svFlags & SV_FLAG_CHEATS) ) { //code }